react-validated-input-component/dist/styles.css

```diff
@@ -0,0 +1,25 @@
+body {
+  font-family: sans-serif;
+}
+
+label {
+    display: block;
+    margin-bottom: 0.5em;
+}
+
+i {
+  margin-left: 0.5em;;
+}
+
+p {
+  font-size: 0.9em;
+  color: red;
+}
+
+.fa-times {
+  color: red;
+}
+
+.fa-check {
+  color: green;
+}
```

Comment:
As mentioned in the other PR, we don't want to change the style of classes from a css library. Here you would want to do something like:

```CSS
.icon-red {
  color: red;
}

.icon-green {
  color: green;
}
```